src/Features/Core/Portable/SolutionCrawler/IncrementalAnalyzerBase.cs

```diff
@@ -67,5 +64,9 @@ public void LogAnalyzerCountSummary()
         /// Order all incremental analyzers below DiagnosticIncrementalAnalyzer
         /// </summary>
         public int Priority => 1;
+
+        public virtual void Shutdown()
```

Comment:
does anyone override this?  i'm not seeing it.

Reply:
consider making non-virtual if that's the case.

Reply:
Nobody overrides. Some of the other virtual methods of this type are also not overridden.